src/mpyl/schema/mpyl_config.schema.yml

```diff
@@ -134,24 +138,48 @@ definitions:
         description: >-
           The path to the folder where the rendered helm chart or kubectl manifest will be written.
           Defaults to target/kubernetes under the root folder of the project.
-      rancher:
+      defaultCluster:
         type: object
-        additionalProperties: false
+        description: Default cluster to deploy to. Can be overriden in the project.yaml
         properties:
-          cluster:
-            "$ref": "#/definitions/RancherClusters"
-    required: [ deploymentStrategy ]
-  RancherClusters:
+          test:
+            type: string
+          acceptance:
+            type: string
+          production:
+            type: string
+      clusters:
+        type: array
+        items:
+          "$ref": "#/definitions/NamedCluster"
+  Clusters:
     type: object
     additionalProperties: false
     properties:
       test:
-        "$ref": "#/definitions/RancherCluster"
+        "$ref": "#/definitions/Cluster"
       acceptance:
-        "$ref": "#/definitions/RancherCluster"
+        "$ref": "#/definitions/Cluster"
       production:
-        "$ref": "#/definitions/RancherCluster"
-  RancherCluster:
+        "$ref": "#/definitions/Cluster"
+  NamedCluster:
```

Comment:
Am I missing something or is this change not reflected in the example config?

Reply:
what do you mean?

Reply:
Nvm, I was looking at it wrong!